server/src/test/java/com/cloud/alert/AlertManagerImplTest.java

```diff
@@ -45,7 +53,7 @@ public class AlertManagerImplTest {
     AlertManagerImpl alertManagerImplMock;
 
     @Mock
-    AlertDao alertDaoMock;
+    AlertDao _alertDao;
```

Comment:
do we need the old name convention?

Reply:
It's better to have the same name for the field which is in the `AlertManagerImpl` to avoid issues. I don't remember exactly but some tests were failing earlier when the field name was different in the test file.